test/e2e/auction/index.test.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-undef */
+describe('Fledge', () => {
+	describe('runAdAuction', () => {
+		beforeEach(async () => {
+			await page.goto('http://localhost:3000/test/e2e/');
+		});
+
+		it('should error when no parameters sent', async () => {
+			const fledge = await page.evaluate(() => window.fledge);
+			expect(() => fledge.joinAdInterestGroup()).toThrow();
+		});
+
+		it('should return token when provided minimum required params and valid interest groups', async () => {
+			await page.evaluate(() => window.fledge.joinAdInterestGroup({
+				owner: 'magnite.com',
+				name: 'test-interest',
+				bidding_logic_url: 'http://localhost:3000/test/e2e/mock/bl.js',
+			}, 100000));
```

Comment:
This also seems like leftovers from the `joinAdInterestGroup` rather than the auction tests.